src/python/cli_new/lib/cli/mesos.py

```diff
@@ -114,25 +114,25 @@ def get_container_id(task):
         " Please try again.")
 
 
-def get_tasks(master):
+def get_tasks(master, query=None):
     """
     Get the tasks in a Mesos cluster.
     """
     endpoint = "tasks"
     key = "tasks"
 
     try:
-        data = http.get_json(master, endpoint)
+        data = http.get_json(master, endpoint, query)
     except Exception as exception:
         raise CLIException(
-            "Could not open '/{endpoint}' on master: {error}"
-            .format(endpoint=endpoint, error=exception))
+            "Could not open '/{endpoint}' with query: {query} on master: {error}"
+            .format(endpoint=endpoint, query=query, error=exception))
```

Comment:
This would print `"Could not open '/tasks' with query: None on master: error message"` if `query` is `None`. 

I think it would be cleaner to print `"Could not open '/tasks' with query: {} on master: error message"`, `"Could not open '/tasks' with query parameters: None on master: error message"`, or `"Could not open '/tasks' on master: error message"` if there is no query. 

This would require to check for the content of query before raising the exception.